integration/kubernetes/helm-chart/alluxio/values.yaml

```
@@ -194,25 +194,35 @@ jobMaster:
 # UFS journal with HDFS example
 # journal:
 #   type: "UFS"
+#   ufsType: "non-local"
 #   folder: "hdfs://{$hostname}:{$hostport}/journal"
 # EMBEDDED journal to /journal example
 # journal:
 #   type: "EMBEDDED"
 #   folder: "/journal"
 journal:
-  type: "UFS" # "UFS" or "EMBEDDED"
-  ufsType: "local" # Ignored if type is "EMBEDDED". "local" or "HDFS"
-  folder: "/journal" # Master journal folder
+  # [ Required arguments ]
```

nit: I could be wrong, but maybe properties/configurations are better here?
```suggestion
  # [ Required properties ]
```